src/backend/marsha/core/tests/test_command_clean_medialive_dev_stack.py

```diff
@@ -0,0 +1,311 @@
+        ) as delete_mediapackage_channel_mock, mock.patch(
+            "django.utils.timezone.now",
+            return_value=datetime(2021, 8, 26, 13, 25, tzinfo=timezone.utc),
+        ), Stubber(
+            clean_medialive_dev_stack.medialive_client
+        ) as medialive_client_stubber:
+            list_medialive_channels_mock.return_value = [
+                {
+                    # non dev environment are ignored
+                    "Name": "production_333c8b00-bb21-44e8-b1b4-286f7e83dd2e_1629984950",
+                    "Tags": {
+                        "app": "marsha",
+                        "environment": "production",
+                    },
+                },
+                {
+                    "Name": "dev-foo_2389cabd-fc46-4ead-8f82-610f35c37d63_1629984950",
+                    "Tags": {"app": "marsha", "environment": "dev-foo"},
+                },
+                {
+                    "Name": "dev-bar_99d60314-20f2-4847-84a4-d2f47bf7fe38_1629982356",
+                    "Tags": {
+                        "app": "marsha",
+                        "environment": "dev-bar",
+                    },
+                    "Id": "562345",
+                    "State": "RUNNING",
+                    "InputAttachments": [
+                        {
+                            "InputId": "876294",
+                        }
+                    ],
+                },
+            ]
+
+            medialive_client_stubber.add_response(
+                "stop_channel",
+                expected_params={"ChannelId": "562345"},
+                service_response={},
+            )
+            medialive_client_stubber.add_response(
+                "describe_channel",
+                expected_params={"ChannelId": "562345"},
+                service_response={"State": "IDLE"},
+            )
+            medialive_client_stubber.add_response(
+                "delete_channel",
+                expected_params={"ChannelId": "562345"},
+                service_response={},
+            )
+
+            medialive_client_stubber.add_response(
+                "describe_input",
+                expected_params={"InputId": "876294"},
+                service_response={"State": "DETACHED"},
+            )
+            medialive_client_stubber.add_response(
+                "delete_input",
+                expected_params={"InputId": "876294"},
+                service_response={},
+            )
+
+            call_command("clean_medialive_dev_stack", stdout=out)
+
+            delete_mediapackage_channel_mock.assert_called_with(
+                "dev-bar_99d60314-20f2-4847-84a4-d2f47bf7fe38_1629982356"
+            )
+            medialive_client_stubber.assert_no_pending_responses()
+
+        self.assertIn(
+            "Cleaning stack with name dev-bar_99d60314-20f2-4847-84a4-d2f47bf7fe38_1629982356",
+            out.getvalue(),
+        )
+        self.assertNotIn(
```

Comment:
Maybe more robust to check that the output includes exactly what you expected? With this test a change in a word of the sentence can make the test useless?

Reply:
Good idea.